meetups/praha.yml

```diff
@@ -310,3 +310,12 @@
   end: 2019-12-04
   time: každou středu 18:00 - 20:00
   place: *ntk
+
+- name: PyData kurz pro PyLadies: <br> Navazující kurz datové analýzy
```

Comment:
Odmaž prosím tu dvojtečku v názvu.

Reply:
Aha, to jsem nevěděla, že tam nemůže být, už to vím - díky!

Reply:
Možná by to šlo nějak ošetřit, aby to prošlo přes testy, zas tak se v tom nevyznám. Rychlejší smazat. :-)

Reply:
Když má být v názvu dvojtečka (nebo jiný znak, který v YAMLu uvozuje např. `# komentář`, `[seznam]`, `{slovník}`, `*odkaz`, …), jde dát celý název do uvozovek.